bot/__init__.py

```diff
@@ -233,6 +233,14 @@ async def on_message(self, message):
                     )
                 )
 
+        # filter social invites from unverified members
+        if len(message.author.roles) == 0:
```

Comment:
Users are given the Phish Food role after sending one message iirc. Is this behavior okay?

Reply:
Ah, I didn't know. I'll update accordingly.